src/SFA.DAS.LevyTransferMatching.Domain/Types/ApplicationStatus.cs

```diff
@@ -6,5 +6,6 @@ public enum ApplicationStatus : byte
         Approved = 1,
         Accepted = 3,
         FundsUsed = 4,
+        Withdrawn = 5
```

Comment:
You wont want to use 5 here - see [this lookup values page on Confluence](https://skillsfundingagency.atlassian.net/wiki/spaces/NDL/pages/3344335014/Lookup+values) (this page will need updating as well) - you might want to use 6 instead. You might also need to add the reference data in [this stored procedure](https://github.com/SkillsFundingAgency/das-data-management/blob/9b07ac24d0d99a040b38ac4a6bb9f346c484da11/src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/LoadReferenceData.sql#L109-L114).

Keep in mind to confirm with @chrisfoster76 the above - I know we can't have clashing status values, though!